storybook/src/components/Menu/Menu.docs.mdx

```diff
@@ -15,31 +15,19 @@ import { StatusBadge } from "../../docs/components/StatusBadge.tsx";
 
 <Markdown>{README}</Markdown>
 
+> **Note**
```

Comment:
Bit of a nitpick, but: a `blockquote` is meant for quotes, not to highlight some text. Should we use Spotlight here?